completion_aggregator/api/common.py

```diff
@@ -120,8 +120,11 @@ def user(self):
 
         requested_username = self.request.GET.get('username')
         if not requested_username:
-            user = self.request.user
-            self._requested_user = None
+            if self.request.user.is_staff:
+                user = self.request.user
+                self._requested_user = None
+            else:
+                raise PermissionDenied()
```

Comment:
@Metfriet, you have to add unit tests for this scenario where the user is trying to access the endpoint without providing a username query string parameter.

Reply:
@lgp171188 I've added 2 unit tests for this.